src/petals/models/bloom/config.py

```diff
@@ -0,0 +1,37 @@
+import os
+from typing import Optional, Union
+
+from hivemind import get_logger
+from transformers.models.bloom import BloomConfig
+from transformers.models.bloom.modeling_bloom import BloomAttention
+
+from petals.client.lm_head import LMHeadConfig
+from petals.client.ptune import PTuneConfig
+from petals.client.routing.sequence_manager import SequenceManagerConfig
+from petals.models.bloom.block import WrappedBloomBlock
+from petals.utils.auto_config import AutoDistributedConfig
+from petals.utils.version import get_compatible_model_repo
+
+logger = get_logger(__name__)
+
+
+class DistributedBloomConfig(BloomConfig, SequenceManagerConfig, PTuneConfig, LMHeadConfig):
+    block_class = WrappedBloomBlock
+    attn_class = BloomAttention
+    block_prefix = "h"
+
+    compat_suffix = "-prefix"
```

Comment:
Nitpicking: can you please add a `#comment` why we need a compat_suffix

Reply:
This was not actually used :) Removed it.